static/app/views/alerts/rules/details/header.tsx

```diff
@@ -33,7 +34,7 @@ function DetailsHeader({hasIncidentRuleDetailsError, rule, params}: Props) {
         <AlertBreadcrumbs
           crumbs={[
             {label: t('Alerts'), to: `/organizations/${params.orgId}/alerts/rules/`},
-            {label: t('Alert Rule')},
+            {label: ruleTitle},
```

Comment:
i thought we were doing alert details

Reply:
Nah, I synced with Phillip already about this though.

Reply:
And the design team